pkg/master/thirdparty/thirdparty.go

```diff
@@ -247,9 +247,6 @@ func (m *ThirdPartyResourceServer) InstallThirdPartyResource(rsrc *extensions.Th
 	if err != nil {
 		return err
 	}
-	if len(rsrc.Versions) == 0 {
-		return fmt.Errorf("ThirdPartyResource %s has no defined versions", rsrc.Name)
-	}
```

Comment:
yes, we should keep this, `InstallThirdPartyResource` could be called by other caller.